core/src/test/resources/internal-port-check-dockerfile/Dockerfile-tcp

```diff
@@ -0,0 +1,7 @@
+# If you ever upgrade this image, watch out for bash being installed by default in newer alpine base images
```

Comment:
consider adding `RUN` that will test the absence of `bash` instead of a text warning

Reply:
Sounds good, will do so